<launch>
    <!-- Launch ROS bridge server -->
    <node pkg="rosbridge_server" type="rosbridge_websocket" name="rosbridge_websocket" output="screen" />

    <!-- Launch LiveKit streamer node -->
    <node name="livekit_streamer" pkg="elderly_bot" type="livekit_streamer.py" output="screen" launch-prefix="/opt/livekit_venv/bin/python3.8" />
</launch>